scripts/check_blocking_repos.py

```diff
@@ -179,3 +199,8 @@ def is_released(repo, dist_file):
     print('The following repos can be released, and are blocking other repos:')
     print('\n'.join(
         sorted('\t{0}'.format(repo) for repo in unblocked_blocking_repos)))
+
+if len(invalid_names):
+    print('Could no resolve the following arguments in %s: ' % prev_distro_key)
+    print('\n'.join(
+        sorted('\t{0}'.format(repo) for repo in invalid_names)))
```

Comment:
I was thinking of it as a summary output, but it does make sense to make it an error code and return 1 if it does not resolve all arguments.